flink-mesos/src/main/java/org/apache/flink/mesos/runtime/clusterframework/services/MesosServicesUtils.java

```diff
@@ -40,9 +41,11 @@ public static MesosServices createMesosServices(Configuration configuration) thr
 				return new StandaloneMesosServices();
 
 			case ZOOKEEPER:
-				final String zkMesosRootPath = configuration.getString(
+				final String zkMesosRootPath = ConfigurationUtil.getStringWithDeprecatedKeys(
+					configuration,
 					ConfigConstants.HA_ZOOKEEPER_MESOS_WORKERS_PATH,
-					ConfigConstants.DEFAULT_ZOOKEEPER_MESOS_WORKERS_PATH);
+					ConfigConstants.DEFAULT_ZOOKEEPER_MESOS_WORKERS_PATH,
+					ConfigConstants.ZOOKEEPER_MESOS_WORKERS_PATH);
```

Comment:
I think it's better if we replace this directly by a `ConfigOption`. There you can also define deprecated keys. This is the encouraged way to read configuration values. Take a look at `HighAvailabilityOptions` to see how it is used.